driver/manager.go

```diff
@@ -51,6 +52,8 @@ type InitConfig struct {
 	// This object needs updates to be generic for different drivers
 	Name             string
 	DockerAPI        dockerclient.APIClient
+	BuildkitAPI      *buildkitclient.Client
```

Comment:
This is very driver-specific and shouldn't be here. Only the remote driver itself should return the client. For the address, not exactly sure why it is here. Maybe we need a generic "endpoint" field in here that is buildkit address for remote driver and docker address for container one.